resources/views/partials/sidebar.blade.php

```diff
@@ -0,0 +1,42 @@
+<script defer src="https://use.fontawesome.com/releases/v5.0.8/js/solid.js" integrity="sha384-+Ga2s7YBbhOD6nie0DzrZpJes+b2K1xkpKxTFFcx59QmVPaSA8c7pycsNaFwUK6l" crossorigin="anonymous"></script>
+<script defer src="https://use.fontawesome.com/releases/v5.0.8/js/fontawesome.js" integrity="sha384-7ox8Q2yzO/uWircfojVuCQOZl+ZZBg2D2J5nkpLqzH1HY0C1dHlTKIbpRz/LG23c" crossorigin="anonymous"></script>
```

Comment:
You probably want these global

Reply:
I would also recommend you add them as a dependency through yarn

Reply:
Done. Sadly the package is quite large. I have been looking into a webpack plugin that can apparently exclude all icons that aren't used, but haven't been able to get this working yet.

Reply:
I have found another option, [fontello](http://fontello.com), which allows the creation of a custom font using the icons of other fonts. I think it would be a good alternative.